doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,32 @@
+# InfiniTime Vision
+
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+## InfiniTime
+
+InfiniTime is a community-built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
+
+InfiniTime is not to be used for medical or other health tracking purposes.
+
+## Core Principles
+
+- KISS (Keep It Short and Simple)
+- Reliability
+- Battery efficiency
+- Easy and simple navigation
+- Behaviour should be predictable and easy to understand
+- Prefer solid default experience over customization
+- Personalization is achieved through custom watchfaces.
+  More options may be available through a companion app.
+- Use standard protocols and methods
+
+## Long term vision
+
+The perfect version of InfiniTime would include:
+
+- Capability to upload apps and watchfaces from a phone
+- Only a minimal feature set in the flashed firmware.
+  Users would add the features they want.
+- Ports to other devices
+- Translations
```

Comment:
**Pitfall**: localizing apps is incredibly hard to do, and comes with a lot of unexpected challenges. PineTime requires users to be tech-savy — so they can overwhelmingly understand basic English. Is support for other languages worth it before a commercial pre-installed noob-friendly watch gets released?

Reply:
There isn't a plan currently to implement translations, but it's certainly something that would be great to have someday, hence I think long term vision is the correct place for this.